src/Interpreters/AddAliasToAnonymousSubqueryVisitor.cpp

```diff
@@ -0,0 +1,42 @@
+#include <Interpreters/AddAliasToAnonymousSubqueryVisitor.h>
+
+#include <Parsers/ASTSubquery.h>
+
+namespace DB
+{
+
+void AddAliasToAnonymousSubqueryVisitor::visit(IAST * ast)
```

Comment:
This just a copy-paste of [`QueryAliasesMatcher<>::visit`](https://github.com/ClickHouse/ClickHouse/blob/c683cb252f84bf540067e4f173a1abf056f0881b/src/Interpreters/QueryAliasesVisitor.cpp#L97), maybe put a comment?